[rospy.client][INFO] 2021-12-03 13:14:44,985: [32minit_node, name[/ss42/inverse_kinematics_node], pid[28055][0m
[xmlrpc][INFO] 2021-12-03 13:14:44,989: [32mXML-RPC server binding to 0.0.0.0:0[0m
[xmlrpc][INFO] 2021-12-03 13:14:44,992: [32mStarted XML-RPC server [http://ss42.local:43595/][0m
[rospy.init][INFO] 2021-12-03 13:14:44,993: [32mROS Slave URI: [http://ss42.local:43595/][0m
[rospy.impl.masterslave][INFO] 2021-12-03 13:14:44,994: [32m_ready: http://ss42.local:43595/[0m
[xmlrpc][INFO] 2021-12-03 13:14:44,996: [32mxml rpc node: starting XML-RPC server[0m
[rospy.registration][INFO] 2021-12-03 13:14:45,005: [32mRegistering with master node http://ss42.local:11311/[0m
[rospy.init][INFO] 2021-12-03 13:14:45,095: [32mregistered with master[0m
[rospy.rosout][INFO] 2021-12-03 13:14:45,096: [32minitializing /rosout core topic[0m
[rospy.rosout][INFO] 2021-12-03 13:14:45,121: [32mconnected to core topic /rosout[0m
[rospy.simtime][INFO] 2021-12-03 13:14:45,132: [32m/use_sim_time is not set, will not subscribe to simulated time [/clock] topic[0m
[rosout][WARNING] 2021-12-03 13:14:45,174: [33m[/ss42/inverse_kinematics_node] /home/robotikabot/duckiefleet/calibrations/kinematics/ss42.yaml does not exist. Using default.yaml.[0m
[rosout][INFO] 2021-12-03 13:14:45,299: [32m[/ss42/inverse_kinematics_node] ~gain = 1.0 [0m
[rosout][INFO] 2021-12-03 13:14:45,327: [32m[/ss42/inverse_kinematics_node] ~trim = 0.0 [0m
[rosout][INFO] 2021-12-03 13:14:45,354: [32m[/ss42/inverse_kinematics_node] ~baseline = 0.1 [0m
[rosout][INFO] 2021-12-03 13:14:45,375: [32m[/ss42/inverse_kinematics_node] ~radius = 0.0318 [0m
[rospy.internal][INFO] 2021-12-03 13:14:45,387: [32mtopic[/rosout] adding connection to [/rosout], count 0[0m
[rosout][INFO] 2021-12-03 13:14:45,404: [32m[/ss42/inverse_kinematics_node] ~k = 27.0 [0m
[rosout][INFO] 2021-12-03 13:14:45,430: [32m[/ss42/inverse_kinematics_node] ~limit = 1.0 [0m
[rosout][INFO] 2021-12-03 13:14:45,536: [32m[/ss42/inverse_kinematics_node] Initialized.[0m
[rosout][INFO] 2021-12-03 13:14:45,543: [32m[/ss42/inverse_kinematics_node] gain: 1.0 trim: 0.0 baseline: 0.1 radius: 0.0318 k: 27.0 limit: 1.0[0m
[rospy.internal][INFO] 2021-12-03 13:14:45,551: [32mtopic[/ss42/joy_mapper_node/car_cmd] adding connection to [http://ss42.local:42021/], count 0[0m
[rospy.internal][INFO] 2021-12-03 13:14:45,610: [32mtopic[/ss42/wheels_driver_node/wheels_cmd] adding connection to [/ss42/wheels_driver_node], count 0[0m
[rospy.core][INFO] 2021-12-03 13:16:33,128: [32msignal_shutdown [signal-2][0m
[rospy.internal][ERROR] 2021-12-03 13:16:33,278: [31mUnable to initiate TCP/IP socket to ss42.local:42701 (http://ss42.local:42021/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 650, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 593, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down
[0m
[rospy.internal][INFO] 2021-12-03 13:16:33,324: [32mtopic[/ss42/wheels_driver_node/wheels_cmd] removing connection to /ss42/wheels_driver_node[0m
[rospy.internal][INFO] 2021-12-03 13:16:33,329: [32mtopic[/rosout] removing connection to /rosout[0m
[rospy.internal][INFO] 2021-12-03 13:16:33,331: [32mtopic[/ss42/joy_mapper_node/car_cmd] removing connection to http://ss42.local:42021/[0m
[rospy.impl.masterslave][INFO] 2021-12-03 13:16:33,332: [32msignal-2[0m
[rospy.core][INFO] 2021-12-03 13:16:33,802: [32msignal_shutdown [atexit][0m
